core/block_svg.ts

```diff
@@ -107,7 +107,7 @@ export class BlockSvg
   private warningTextDb = new Map<string, ReturnType<typeof setTimeout>>();
 
   /** Block's mutator icon (if any). */
-  mutator: Mutator | null = null;
```

Comment:
This should be deprecated.

Reply:
I was in the middle of trying to get the comment icon refactored :P so I just left myself a note.